mlflow/metrics/utils/prompt_template.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+
+class PromptTemplate:
```

Comment:
Is this a public API?

Reply:
Nope. This won't be a public API. It would be used internally